### DISCUSSION TITLE
After migration from version 9 to 17 the form collapsed

### DISCUSSION THREAD

**Discussion by evgushchin** [NG-ZORRO/ng-zorro-antd#8525] [Q&A] [answered]
I migrated a project from very old version 9 to the latest 17. The styles seem to be in order but the form is collapsed. 
It seems that the style `ant-form-item` is not applied to the `nz-form-item` element.
<img width="455" alt="Screenshot 2024-04-25 at 19 24 39" src="https://github.com/NG-ZORRO/ng-zorro-antd/assets/72899723/5baf998c-ce83-404e-851b-22676fb0a3da">

**Comment by ParsaArvanehPA** [comment, marked as answer]
Greeting @evgushchin ,
In the previous version,you might have used the ::ng-deep pseudo-class to customize the default styles. Following the upgrade, certain class names or tag names might have been changed, which could lead to changes in the appearance of the styles.

To address this, please use the developer tool to confirm whether the ng-zorro styles are being applied as expected. Additionally, check for any custom styles that might be overriding the default ones.

**Comment by evgushchin** [comment]
I just found the issue. I was importing components instead of modules.
